### PR TITLE
Move getContatenadedSignature (PendingMessage -> PendingMessages)

### PR DESCRIPTION
**Pull request by xitronix**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [x] Change is small and easy to review (please split big changes into multiple PRs)
- [x] Change is consistent with architecture
- [x] Change is consistent with test architecture
- [x] Change is consistent with naming conventions
- [x] New code is covered with tests
- [x] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-relayer/lib/services/messages/PendingMessages.ts [hunk 1]
```diff
@@ -45,7 +45,11 @@ export default class PendingMessages {
   }
 
   getMessageWithSignatures(message: SignedMessage, hash: string) : SignedMessage {
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/services/messages/PendingMessages.ts]: rename `hash` to `messageHash`

File: universal-login-relayer/lib/services/messages/PendingMessages.ts [hunk 2]
```diff
@@ -45,7 +45,11 @@ export default class PendingMessages {
...
-    return  { ...message, signature: this.messagesStore.get(hash).getConcatenatedSignatures()};
+    const collectedSignatures = this.messagesStore.getCollectedSignatures(hash);
+    const sortedExecutions = sortExecutionsByKey([...collectedSignatures]);
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/services/messages/PendingMessages.ts]: rename 
- `sortedExecutions` to `sortedSignaturesKeyPair`
- `sortExecutionsByKey` to `sortSignaturesKeyPair`

File: universal-login-relayer/lib/services/messages/PendingMessages.ts [hunk 3]
```diff
@@ -45,7 +45,11 @@ export default class PendingMessages {
...
+    const sortedSignatures = sortedExecutions.map((value: CollectedSignature) => value.signature);
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/services/messages/PendingMessages.ts]: rename
- `sortedExecutions` to `sortedSignatureKeyPairs`
- **`CollectedSignature` to `CollectedSignatureKeyPair`**

File: universal-login-relayer/lib/services/messages/PendingMessagesStore.ts [hunk 4]
```diff
@@ -56,6 +56,10 @@ export default class PendingMessagesStore implements IPendingMessagesStore {
     };
   }
 
+  getCollectedSignatures(messageHash: string) {
+    return this.messages[messageHash].collectedSignatures;
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/services/messages/PendingMessagesStore.ts]: rename
- `getCollectedSignatures` to `getCollectedSignatureKeyPairs`
- `collectedSignatures` to `collectedSignatureKeyPairs`

File: universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts [hunk 5]
```diff
@@ -8,5 +8,12 @@ export default interface IPendingMessagesStore {
   remove: (messageHash: string) => PendingMessage;
   getStatus: (messageHash: string) => Promise<MessageStatus>;
   addSignature: (messageHash: string, signature: string) => void;
+  getCollectedSignatures: (messageHash: string) => CollectedSignature[];
   updateTransactionHash: (messageHash: string, transactionHash: string) => void;
+  containSignature: (messageHash: string, signature: string) => boolean;
 }
+
+export type CollectedSignature = {
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts]: rename from `CollectedSignature` to `CollectedSignatureKeyPair`

File: universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts [hunk 6]
```diff
@@ -8,5 +8,12 @@ export default interface IPendingMessagesStore {
   remove: (messageHash: string) => PendingMessage;
   getStatus: (messageHash: string) => Promise<MessageStatus>;
   addSignature: (messageHash: string, signature: string) => void;
+  getCollectedSignatures: (messageHash: string) => CollectedSignature[];
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts]: `getCollectedSignatures` -> `getCollectedSignatureKeyPairs`

Comment (marekkirejczyk) [reply universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts]: Why? Przecież zwraca `CollectedSignature`?

File: universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts (hunk 5 above)
Comment (JustynaBroniszewska) [reply universal-login-relayer/lib/services/messages/IPendingMessagesStore.ts]: @marekkirejczyk

**Pull request merged**